synapse/tests/test_cortex.py

```diff
@@ -4331,7 +4332,7 @@ async def test_storm_tagvar(self):
             self.nn(nodes[0].getTag('tag1'))
             self.nn(nodes[0].getTag('tag3'))
 
-            q = '$t1="" $t2="" $t3=tag3 [test:str=x -#$t1 +?#$t2 +?#$t3]'
+            q = '$t2="" $t3=tag3 [test:str=x +?#$t2 +?#$t3]'
```

Comment:
subtle compatibility change - the old storm code which did run now raises an exception